spec/eof.md

```diff
@@ -348,6 +381,40 @@ During scanning, for each instruction:
 
 Annotated examples of EOF formatted containers demonstrating several key features of EOF can be found in [this test file within the `evmone` project repository](https://github.com/ethereum/evmone/blob/master/test/unittests/eof_example_test.cpp).
 
+## Appendix: Creator Contract
+
+```solidity
+{
+    /// Takes [tx_initcode_hash][salt][init_data] as input,
+    /// creates contract and returns the address or failure otherwise
+
+    // init_data.length can be 0, but the first 2 words are mandatory
+    let size := calldatasize()
+    if lt(size, 64) { revert(0, 0) }
+
+    let tx_initcode_hash := calldataload(0)
+    let salt := calldataload(32)
+
+    mstore8(0, 0xff)  // a magic value to ensure a specific preimage space
```

Comment:
I'm not convinced about the necessity of extra magic value here, there's already another `0xff` prepended in final hashing of TXCREATE itself, isnt't that enough?

Reply:
I'm not convinced about this either.

Reply:
Same here, let's drop, unless objections arise

Reply:
Done, please take a look if that went well